apis/kueue/v1beta1/workload_types.go

```diff
@@ -329,6 +329,12 @@ const (
 	WorkloadEvictedByDeactivation = "InactiveWorkload"
 )
 
+const (
+	// WorkloadRequeuedByReactivation indicates that the workload was requeued
+	// because spec.active is set to true.
+	WorkloadRequeuedByReactivation = "ReactivateWorkload"
```

Comment:
```suggestion
	WorkloadReactivated = "WorkloadReactivated"
```
It is a `reason`. It will be clear that it is `Requeued` from the condition type.

Reply:
Done